growth/too/models.py

```diff
@@ -785,6 +785,15 @@ def start_observation(self):
         else:
             return None
 
+    @property
+    def end_observation(self):
+        """Time of the end of planned observations."""
+        if self.planned_observations:
+            end = Time(self.start_observation).jd + self.tot_time_with_overheads/86400.0
+            return Time(end, format='jd').datetime
```

Comment:
This should return a `datetime.datetime`, like its sibling property, `start_observation`.

Reply:
Yes, I tested the code and can confirm that both are in datetime.datetime format.

Reply:
OK, but you are converting from datetime.datetime (the type that comes out of SQLAlchemy) to astropy.time.Time, to float (jd), to astropy.time.Time, to datetime.datetime. No conversions are necessary here.